src/components/Modal/Modal.stories.tsx

```diff
@@ -206,7 +206,9 @@ export const ContentDefault: Story = {
     children: (
       <>
         <Modal.Header>
-          <Heading as="h2">Modal Title</Heading>
+          <Heading as="h2" className="text-utility-default-primary">
```

Comment:
This `Heading` component is used in these tests to work around a problem in HeadlessUI and storybook..

* this should be `Modal.Title`
* HeadlessUI doesn't allow for the underlying `.Title` component to be outside the parent
* Since our `Modal` wrapper uses `Heading` under the hood, we use it here to display the open modal in stories

A follow-up task is to make this abundantly clear in the documentation.